internal/boxcli/run.go

```diff
@@ -41,6 +43,30 @@ func RunCmd() *cobra.Command {
 	return command
 }
 
+func InstallCmd() *cobra.Command {
+	flags := runCmdFlags{}
+	command := &cobra.Command{
+		Use:   "install",
+		Short: "Installs all packages mentioned in devbox.json",
+		Long: "Starts a new shell and installs all packages mentioned in devbox.json or a json file" +
```

Comment:
Should we clarify that the shell is a devbox shell?
i.e. suggestion: "Starts a new `devbox shell` and ...."